src/apd/util.go

```diff
@@ -0,0 +1,79 @@
+package apd
+
+import (
+	"bytes"
+	"encoding/gob"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/SkycoinProject/skycoin/src/util/logging"
+)
+
+var logger = func(moduleName string) *logging.Logger {
+	masterLogger := logging.NewMasterLogger()
+	return masterLogger.PackageLogger(moduleName)
+}
+
+const moduleName = "apd.broadcast"
+
+// BroadCastPubKey broadcasts a UDP packet containing the public key of the local visor.
+// Broadcasts is sent on the local network broadcasts address.
+func BroadCastPubKey(pubkey, broadCastIP string, port int) error {
+	address := fmt.Sprintf("%s:%d", broadCastIP, port)
+	bAddr, err := net.ResolveUDPAddr("udp", address)
+	if err != nil {
+		logger(moduleName).Errorf("Couldn't resolve broadcast address: %v", err)
+		return err
+	}
+
+	conn, err := net.DialUDP("udp", nil, bAddr)
+	if err != nil {
+		return err
+	}
+
+	defer conn.Close()
+
+	packet := []byte(pubkey)
+	_, err = conn.Write(packet)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func Deserialize(data []byte) (Packet, error) {
```

Comment:
It seems to be unused. Can we delete it?

Reply:
Deleted